man/ocitools-validate.1.md

```diff
@@ -18,6 +21,9 @@ Validate an OCI bundle
 **--path=PATH
    Path to bundle
 
+**--hooks-check**
```

Comment:
I [still prefer][1] the more-explicit `--hook-paths` (but I'm not a maintainer).  Whatever the option name ends up being, I'd advise against including `-check` (so just `--hooks` if you don't want an option that mentions paths).  I'd guess almost all future options to `validate` are going to be about enabling/disabling particular checks (since checking fields is all `validate` does ;).  And in that case, the fact that a hook enables/disables a check is pretty obvious even without the `-check` qualifier.

[1]: https://github.com/opencontainers/ocitools/pull/49#discussion_r62611097